Kerbee/Functions/PurgeExpiredKeysActivity.cs

```diff
@@ -26,7 +26,7 @@ public async Task<object> RunAsync([ActivityTrigger] Application application)
         }
         catch (Exception ex)
         {
-            _logger.LogError(ex, "Error purging keys for application {ApplicationId}", application.Id);
+            _logger.LogError(ex, "Error purging keys for application {ApplicationId}", application.AppId);
```

Comment:
Waarom zou je try/catchen en dan gewoon throwen? Als je dat niet doet zie je toch ook de logs?

Reply:
Deze change heeft Sander vorige maand expliciet er in gezet:
https://github.com/WeAreInSpark/key-rotation-bot/pull/16/files